products/mcp/typescript/src/resources/integration/index.ts

```diff
@@ -129,11 +122,27 @@ export function registerIntegrationResources(server: McpServer, _context: Contex
                 description: workflow.description,
             },
             async (uri) => {
+                // Fetch the examples ZIP which includes prompts (from cache if available)
+                const examplesZip = await fetchExamplesMarkdown()
+
+                // Read the workflow file from the ZIP
+                const promptData = examplesZip[workflow.file]
+                if (!promptData) {
```

Comment:
Curious, how does this error get handled by the consumers of the MCP? Is it throw that far down at all or is this just for our logging and error tracking?